plugins/module_utils/common/api/PROPERTY_STYLE_COMPARISON.md

```diff
@@ -0,0 +1,299 @@
+# Property-Style Interface: Constructor vs Property-Based
+
```

Comment:
Let's remove these before merge.  We have the commit history and can refer to it later if needed.